mathics/builtin/assignments/clear.py

```diff
@@ -167,6 +167,34 @@ def do_clear(self, definition):
         definition.defaultvalues = []
 
 
+class Remove(Builtin):
+    """
+    <dl>
+    <dt>'Remove[$x$]'
+        <dd>removes the definition associated to $x$
+    </dl>
+    >> a := 2
+    >> Names["Global`a"]
+     = {a}
+    >> Remove[a]
+    >> Names["Global`a"]
+     = {}
+    """
+
+    attributes = A_HOLD_ALL | A_LOCKED | A_PROTECTED
+    messages = {"ssym": "`1` is not a symbol."}
```

Comment:
I see "ssym" defined a number of places: 

```
PAGER=cat git grep -n "ssym"
mathics/builtin/assignments/clear.py:80:        "ssym": "`1` is not a symbol or a string.",
mathics/builtin/assignments/clear.py:114:                    evaluation.message(self.get_name(), "ssym", symbol)
mathics/builtin/attributes.py:209:        "ssym": "`1` is not a symbol or a string.",
mathics/builtin/attributes.py:226:                evaluation.message("Protect", "ssym", symbols)
mathics/builtin/attributes.py:235:                    evaluation.message("Protect", "ssym", symbol)
mathics/builtin/attributes.py:265:        "ssym": "`1` is not a symbol or a string.",
mathics/builtin/attributes.py:288:                    evaluation.message("Unprotect", "ssym", symbol)
```

I think if we just add it to `mathics-core/mathics/builtin/messages.py` we can DRY the code:

Reply:
Yep.

Reply:
Let's do it in the next round.